src/test/java/uk/gov/hmcts/cmc/claimstore/rules/CountyCourtJudgmentRuleTest.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.cmc.claimstore.rules;
+
+import org.junit.Test;
+import uk.gov.hmcts.cmc.claimstore.exceptions.ForbiddenActionException;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaim;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+public class CountyCourtJudgmentRuleTest {
+
+    private final CountyCourtJudgmentRule countyCourtJudgmentRule = new CountyCourtJudgmentRule();
+
+    @Test
+    public void saveShouldFinishSuccessfullyForHappyPath() {
```

Comment:
That does not mean anything to me, please describe like `shouldNotThrowExceptionWhenCCJCanBeRequested`

Reply:
done